foundation-ui/mount/MountService.js

```diff
@@ -0,0 +1,94 @@
+import {EventEmitter} from 'events';
+import {CHANGE} from './MountEvent';
+
+/**
+ * List of component descriptors
+ *
+ * @type {Array.<{component:function, priority:Number, type:String}>}
+ */
+const components = [];
```

Comment:
Agreed, performance is probably marginal, it's more that it made more sense in my head to do it this other way. Cool with deferring.